<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mongodb_gui_documents</class>
 <widget class="QWidget" name="mongodb_gui_documents">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>406</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MongoDB:  Document Managent</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="GeneralLayout">
     <item>
      <layout class="QHBoxLayout" name="SelectionPanel">
       <item>
        <layout class="QVBoxLayout" name="databaseManagementPanel">
         <item>
          <widget class="QLabel" name="databaseLabel">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Database</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="database_comboBox"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="databaseManagementControlers">
           <item>
            <widget class="QPushButton" name="addDatabaseButton">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteDatabaseButton">
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="collectionManagementPanel">
         <item>
          <widget class="QLabel" name="collectionLabel">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Collection</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="collection_comboBox"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="CollectionManagementControlers">
           <item>
            <widget class="QPushButton" name="exportButton">
             <property name="text">
              <string>Export </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addCollectionButton">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteCollectionButton">
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="selectionPanel">
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="selectButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="DocumentManagementPanel">
       <item>
        <layout class="QHBoxLayout" name="DocumentAndContentLabels">
         <item>
          <widget class="QLabel" name="ContentLabel">
           <property name="text">
            <string>Content</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="DocumentLabel">
           <property name="text">
            <string>Documents</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="DocumentsListAndFileContent">
         <item>
          <widget class="QListWidget" name="documentList"/>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="fileContentTextBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="SubPanel">
         <item>
          <layout class="QVBoxLayout" name="DocumentManagementControlers">
           <item>
            <widget class="QLabel" name="DocumentOptionsLabel">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Document options</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="ButtonsPanel">
             <item>
              <widget class="QPushButton" name="uploadButton">
               <property name="text">
                <string>Upload</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="downloadButton">
               <property name="text">
                <string>Download</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="deleteButton">
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
